### DISCUSSION TITLE
Application Stuck "processing" months

### DISCUSSION THREAD

**Discussion by tgriffiths04** [community/community#122925] [GitHub Education]
### Hi everyone,

Hi all,

I'm not too sure what to do with this, it seems to say that my application is processing, however I have heard nothing about it at all, and it lets me start a new application, however the submit button does not seem to do anything, I have received no feedback via email, is this expected as it has been close to 2 months?

**Comment by tgriffiths04** [comment]
If it is of any relevance, the process application button seems to just clear the application, and does not give me any response

**Comment by davevad93** [comment]
Hi @tgriffiths04 ,

If you're talking about the application to obtain your students benefits, it is strange indeed that it has been stuck for 2 months.
Maybe you should directly ask this at [GitHub Support](https://support.github.com/request/education) , I hope you can solve it.
